### ISSUE TITLE
Keyboard shortcuts that use the `access` modifier don't work with VoiceOver

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#11154]
Splitting this out from #5694 see #10545 

Gutenberg uses a convention for keyboard shortcuts where the `access` combination translates, on macOS, to `Control Option`. Currently, there are a few shortcuts that use this modifier, including the new Blocks navigation menu implemented in #10545.

Unfortunately, VoiceOver on macOS uses `Control Option` as its own default modifier so all the Gutenberg shortcuts that use `Control Option` either trigger a specific VoiceOver feature or don't work at all.

So far, I've found the following shortcuts using `Control Option`:

```
Display this help. ^ ⌥ H
Open the block navigation menu. ^ ⌥ O
Remove the selected block(s). ^ ⌥ Z
Remove a link. ^ ⌥ S
Add a strikethrough to the selected text. ^ ⌥ D
Display the selected text in a monospaced font. ^ ⌥ X
Add link ^ ⌥ a (undocumented in the Help Modal, alternative shortcut available)
```

Basically, all the shortcuts implemented with `rawShortcut.access()` or `<Shortcut type="access" ...`. All of these need to be changed to use a different modifier.

Aside: still thinking the best path forward is a mechanism to allow users to re-map and customize shortcuts, as proposed in #3218.

**Comment by iseulde** [comment]
So this is also an issue with the old editor? WordPress has used the "access" modifier since a long time, and it was hard to find anything that didn't clash with the OS or browser. Has this always been the same in VoiceOver?

**Comment by afercia** [comment]
Correct.
For example, testing with Safari + VoiceOver, `Ctrl-Alt-{1-6}` to change a heading level doesn't work.

**Comment by afercia** [comment]
See related core Trac ticket: https://core.trac.wordpress.org/ticket/39271

**Comment by afercia** [comment]
For reference: in the new WAI-ARIA Authoring Practices 1.1
there's a pretty good section dedicated to keyboard shortcuts. It's more about what not to do:
https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_shortcuts_assigning

Among others, avoid: 
> macOS only: Control+Option + any combination of other keys.

**Comment by tbdalgaard** [comment]
Most of the issue here could be fixed if option was removed so the keystroke would be ctrl followed by the letter/number instead. That would solve the issues on Mac OS with Safari.

**Comment by ellatrix** [comment]
Is there an alternative combination that can work, like the one for Windows, that doesn't involve more than 2 modifiers?
